src/components/card/MovieCard.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react'
+import classNames from 'classnames'
 
 interface Props {
   title: string
-  posterPath: string
-  onClick: (title: string) => void
+  posterPath?: string
+  onClick?: (title: string) => void
 }
 
 const MovieCard = ({ title, posterPath, onClick }: Props) => {
```

Comment:
These changes make the `MovieCard` more generic and usable on the bonus screen as well